### PR TITLE
Push video incoming call

### PR DESCRIPTION
**Pull request by corlard3y**
## What does this PR do?

copilot:summary

## Related issues

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Explanation of the changes

copilot:walkthrough

## Emoji

copilot:emoji

### CODE REVIEW

**Review by madhur-push — CHANGES_REQUESTED** [review]
- Please rename the whitevideobtn.svg to callacceptbtn.svg
- The minimized version has some problems with positioning on widths less than 1440px, please check this once. Also, the button icon (accept reject) widths has problems for mobiles devices (check at 375px and below).
- Its such that upon an outside click of the modal we dont want the modal to disappear rather on outside click we can trigger the minimized version.

File: packages/ui/src/Modal.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ interface ModalProps {
   show: boolean;
   children: ReactNode[] | ReactNode;
   dataTestId?: string;
+  darkModeBgColor?: string;
```
Comment (0xNilesh) [inline packages/ui/src/Modal.tsx:14]: Don't try to change anything in their design system or their shared folder
even here, I don't see any props related to css, so it's not ideal to add this css prop just for a dark mode color

if it's very necessary to do it, then create a new
but if change can be ignored, then leave it
here, we can leave this change, and keep our modal similar their modals unless this is very necessary to put

Comment (corlard3y) [reply packages/ui/src/Modal.tsx:14]: understood, their dark mode bg colors have a different color from ours according to design, do we ask design to adjust it or make a new reusable modal in our own folders?

Comment (0xNilesh) [reply packages/ui/src/Modal.tsx:14]: ask design team to match it to theirs if possible
if not, then we make our own

**Review by madhur-push — APPROVED** [review]
LGTM

**Pull request merged**